sdk/python/feast/repo_config.py

```diff
@@ -244,6 +279,15 @@ def get_offline_config_from_type(offline_store_type: str):
     return get_class_from_type(module_name, config_class_name, config_class_name)
 
 
+def get_feature_server_config_from_type(feature_server_type: str):
+    # We do not support custom feature servers right now.
+    assert feature_server_type in FEATURE_SERVER_CONFIG_CLASS_FOR_TYPE
```

Comment:
Ideally we'd avoid `assert` usage since asserts can be disabled by an optimizer.

Reply:
Got it, should we just raise a custom error defined in `errors.py` instead? cc @woop

Reply:
In my opinion, there's no need to create custom errors for super simple stuff. I'd recommend throwing `ValueError` on validations. In this particular case you can simply remove this line since `FEATURE_SERVER_CONFIG_CLASS_FOR_TYPE[feature_server_type]` will fail anyways.